### DISCUSSION TITLE
Create new indexes in Elasticsearch and mapping - RUMv2

### DISCUSSION THREAD

**Discussion by Mato816** [canarytrace/documentation#204] [RUM]
Hi @rdpanek ,

I would like to create new indexes and map them for RUMv2.
I found this https://github.com/canarytrace/documentation/issues/120 manual how to complete it.

When I run these docker commands:
`docker run --rm -it --entrypoint /bin/mv -v $(pwd):/postman quay.io/canarytrace/rum:1.10 /opt/canary-rum/postman/ /postman/`

`docker run --rm -it --entrypoint /bin/mv -v $(pwd):/elastic_kibana quay.io/canarytrace/rum:1.10 /opt/canary-rum/elastic_kibana/ /elastic_kibana/`

I got error:
```
Unable to find image 'quay.io/canarytrace/rum:1.10' locally
docker: Error response from daemon: manifest for quay.io/canarytrace/rum:1.10 not found: manifest unknown: manifest unknown.
```

I tried repeat these commands with tag `2.9.2` and got another error:
`mv: can't rename '/opt/canary-rum/postman': No such file or directory`

`mv: can't rename '/opt/canary-rum/elastic_kibana': No such file or directory`

Can you help me how to download files necessary to create and map new indexes?
Thank you.

**Comment by rdpanek** [comment]
Hello, please follow this documentation https://v2doc.canarytrace.com/docs/rum/kibana#how-to-get-visualizations Thanks
